packages/contracts/deploy/016-Proxy__OVM_L1StandardBridge.deploy.ts

```diff
@@ -0,0 +1,53 @@
+/* Imports: External */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+/* Imports: Internal */
+import { getDeployedContract } from '../src/hardhat-deploy-ethers'
+import { predeploys } from '../src/predeploys'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deploy } = hre.deployments
+  const { deployer } = await hre.getNamedAccounts()
+
+  const Lib_AddressManager = await getDeployedContract(
+    hre,
+    'Lib_AddressManager',
+    {
+      signerOrProvider: deployer,
+    }
+  )
+
+  const result = await deploy('Proxy__OVM_L1StandardBridge', {
```

Comment:
Reminder that this will be a chugsplash proxy and not a `ResolvedDelegateProxy`.